apps/modernization-ui/src/welcomeLayout/welcomeLayout.module.scss

```diff
@@ -23,20 +23,34 @@
         background-color: colors.$base-white;
 
         .content {
-            margin: 1rem;
-            padding: 1rem;
+            margin: 0 1rem;
+            padding: 0 1rem 0.5rem;
             background-color: rgba(colors.$base-white, 0.9);
             .container {
                 column-width: 40ch;
                 column-gap: 2rem;
                 overflow: hidden;
                 padding-top: 1rem;
+                a {
+                    text-decoration: underline;
+                    font-weight: 400;
+                }
                 span {
                     margin-bottom: 1rem;
                 }
+                h2 {
+                    margin-bottom: 0.25rem;
+                    color: colors.$base-darker;
```

Comment:
All of the text on this page should be the same text color which should already be set in the body.

Reply:
Designs say otherwise
<img width="629" alt="image" src="https://github.com/CDCgov/NEDSS-Modernization/assets/124739504/e0e31825-5aac-4b37-b3b4-7fb8e8395ef1">